pkg/api/operators/apis/experiment/v1alpha2/experiment_types.go

```diff
@@ -79,6 +79,9 @@ type ExperimentStatus struct {
 	// Current optimal trial parameters and observations.
 	CurrentOptimalTrial OptimalTrial `json:"currentOptimalTrial,omitempty"`
 
+	// Trials is the total number of trials owned by the experiment.
+	Trials int `json:"trials,omitempty"`
```

Comment:
What do you think about `TotalTrials` name?

Reply:
I think in k8s community, we prefer trials. See also https://github.com/kubernetes/api/blob/master/apps/v1/types.go#L185 and https://github.com/kubernetes/api/blob/master/apps/v1/types.go#L383

Reply:
Ok